awx_collection/test/awx/test_group.py

```diff
@@ -53,7 +52,5 @@ def test_tower_group_idempotent(run_module, admin_user):
     result.pop('invocation')
     assert result == {
         'id': group.id,
-        'group': 'Test Group',
         'changed': False,  # idempotency assertion
-        'state': 'present'
```

Comment:
Is there any reason for having "name" in the returned data when creating, but not when modifying an object which already exists? That looks inconsistent. If you are including keys because they were non-null params, then okay, but that's not a consistent explanation either, because "name" _was_ provided in the invocation.